nomad/core_sched.go

```diff
@@ -482,19 +483,42 @@ OUTER:
 		return nil
 	}
 	c.logger.Debug("node GC found eligible nodes", "nodes", len(gcNode))
+	return c.nodeReap(eval, gcNode)
+}
+
+func (c *CoreScheduler) nodeReap(eval *structs.Evaluation, nodeIDs []string) error {
+	// For pre 0.9.3 clusters, send single deregistration messages
+	version, _ := version.NewVersion("0.9.3")
```

Comment:
I would suggest extracting that to a variable that's easy to grep for, e.g. `minVersionBatchNodeDeregister`.  We do that pattern elsewhere, e.g. `MinVersionPlanNormalization`, `minSchedulerConfigVersion`, ....  Also, you can use `version.Must()` to assert no error.